pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/PolicyName.java

```diff
@@ -52,5 +52,6 @@ public enum PolicyName {
     RESOURCEGROUP,
     ENTRY_FILTERS,
     SHADOW_TOPIC,
-    DISPATCHER_PAUSE_ON_ACK_STATE_PERSISTENT
+    DISPATCHER_PAUSE_ON_ACK_STATE_PERSISTENT,
+    MESSAGE_SIZE
```

Comment:
Is it necessary to add a new PolicyName? That might cause problems unless it is mapped in possible external AuthorizationProviders. Is there a way to not add a new PolicyName?

Reply:
Perhaps causes exceptions in AuthorizationProviders, https://github.com/apache/pulsar/blob/f427c90be6132669f2a048e7a7d925e714de9a02/pulsar-broker-common/src/main/java/org/apache/pulsar/broker/authorization/PulsarAuthorizationProvider.java#L543-L544